go/adbc/driver/flightsql/flightsql_adbc.go

```diff
@@ -827,6 +1035,45 @@ func (c *cnxn) SetOption(key, value string) error {
 	return nil
 }
 
+func (c *cnxn) SetOptionBytes(key string, value []byte) error {
+	return adbc.Error{
+		Msg:  "[Flight SQL] unknown connection option",
+		Code: adbc.StatusNotImplemented,
+	}
+}
+
+func (c *cnxn) SetOptionInt(key string, value int64) error {
+	switch key {
+	case OptionTimeoutFetch:
+		fallthrough
+	case OptionTimeoutQuery:
+		fallthrough
+	case OptionTimeoutUpdate:
+		return c.timeouts.setTimeout(key, float64(value))
```

Comment:
same question as earlier, any reason to use `fallthrough` over just multiple values in the case? 

`case OptionTimeoutFetch, OptionTimeoutQuery, OptionTimeoutUpdate:`